frontend/src/pages/projects/screens/detail/ProjectDetails.tsx

```diff
@@ -95,40 +44,19 @@ const ProjectDetails: React.FC = () => {
           <BreadcrumbItem isActive>{displayName}</BreadcrumbItem>
         </Breadcrumb>
       }
-      loaded
+      loaded={rbacLoaded}
       empty={false}
     >
-      <PageSection
-        id={scrollableSelectorID}
-        hasOverflowScroll
-        isFilled
-        aria-label="project-details-page-section"
-        variant="light"
-      >
-        <GenericSidebar
-          sections={sections.map(({ id }) => id)}
-          titles={ProjectSectionTitles}
-          scrollableSelector={`#${scrollableSelectorID}`}
-          maxWidth={175}
-        >
-          <Stack hasGutter>
-            {sections.map(({ id, component, isEmpty }, index) => (
-              <React.Fragment key={id}>
-                <StackItem
-                  id={id}
-                  aria-label={ProjectSectionTitles[id]}
-                  data-testid="details-page-section"
-                >
-                  {component}
-                </StackItem>
-                {index !== sections.length - 1 && isEmpty && (
-                  <Divider data-testid="details-page-section-divider" />
-                )}
-              </React.Fragment>
-            ))}
-          </Stack>
-        </GenericSidebar>
-      </PageSection>
+      {projectSharingEnabled && allowCreate ? (
```

Comment:
There's also this funny issue with permissions now, I left it unattended we might wanna fix it here or track it to improve it later cc @andrewballantyne @kywalker-rh 
![permission-issue](https://user-images.githubusercontent.com/16117276/233134971-61f0e863-d0e1-4739-b23e-43a9695ecac1.gif)

Reply:
You violate your own permissions -- that's life. Don't play with fire.

Might be worth having a warning when you do that to say "this is you, you're about to have a bad day"

Reply:
Ok, the only slight issue is that if for example you loose permission to the project it takes the time to refresh it to be kicked out of them, should we refresh that every time?

Reply:
@lucferbux different problem -- handled in a different ticket. Permissions and lack of access is not something you need to worry about right now.

Besides, K8s will lock them out as soon as the permission goes away.

Reply:
Perfect, and yes, that's true, users are just seeing the local version, they cannot do anything with it.